index.html

```diff
@@ -1288,11 +1288,42 @@ <h2>DID Subject</h2>
 However, the fully resolved <a>DID document</a> always contains a valid
 <code><a>id</a></code> property. The value of <code><a>id</a></code> in the
 resolved <a>DID document</a> MUST match the <a>DID</a> that was
-resolved, or be populated with the equivalent canonical <a>DID</a> 
-specified by the <a>DID method</a>, which SHOULD be used by the resolving 
+resolved, or be populated with the equivalent canonical <a>DID</a>
+specified by the <a>DID method</a>, which SHOULD be used by the resolving
 party going forward.
       </p>
 
+      <p>
+For certain use cases it might be useful to explicitly state the nature of the
+<a>DID subject</a> (e.g., person, organization, book, web page, data
+structure, abstract concept, etc.). For this, the <code><a>type</a></code>
+property is used.
+      </p>
+
+      <p>
+<a>DID documents</a> MAY include the <code><a>type</a></code> property.
+      </p>
+
+      <dl>
+        <dt><dfn>type</dfn></dt>
+        <dd>
+Where present, the value of <code>type</code> MUST be a
+<a data-cite="INFRA#lists">list</a> where each item in the list is either a
+<a>URI</a> conforming to [[RFC3986]] <em>or</em> a <a
+data-cite="INFRA#string">string</a> which is registered in the
+[[DID-SPEC-REGISTRIES]].
+        </dd>
+      </dl>
+
+      <pre class="example nohighlight" title="Example of type property">
+{
+  "@context": "https://www.w3.org/ns/did/v1",
+  "id": "did:example:21tDAKCERh95uGgKbJNHYp",
+  "type": ["https://schema.org/Person"],
```

Comment:
I'd prefer a potentially less contentious example, perhaps: 
```suggestion
  "type": ["https://schema.org/Book"],
```

Reply:
@brentzundel While I appreciate that "Book" is much less controversial, it's also potentially a much lower-value use case. May I suggest "https://schema.org/Organization"?

Reply:
I think types are a great idea.  I try to give them to all of my linked objects, as it helps to query by type

If a DID is of type Person

And elsewhere we can say `<did:example:21tDAKCERh95uGgKbJNHYp> :created <Timestamp>` that leads to an ambiguity as to what actually was created, as it applies to both the person, and the identifier

Reply:
Can we change this to "inmate" or "terrorist" ?... I prefer the worst case scenarios be addressed up front :)

@melvincarvalho I believe `created` is "did document metadata` and is not returned in the did document itself, although it would be fine for. "JSON-only" did documents which do not care about RDF / triples.